src/components/ExerciseForm.tsx

```diff
@@ -0,0 +1,121 @@
+import React,{ useState } from 'react'
+import exerciseService from "../services/exercises";
+import { useNavigate } from "react-router-dom";
+
+const ExerciseForm = () => {
+
+    const selectedValue = "Easy";
+    const [title, setTitle] = useState("");
+    const [complexity, setComplexity] = useState("");
+    const [textExercise, setTestExercise] = useState("");
```

Comment:
These values should come from the props so we can reuse this component in create and edit page